dev/com.ibm.ws.security.wim.adapter.ldap/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -170,6 +170,15 @@
 		<AD id="timestampFormat" name="%timestampFormat" description="%timestampFormat.desc" required="false" type="String" />
 		<!-- End VMM ported Entries -->	
 
+	<AD id="bindAuthMechanism" name="internal" description="internal use only" required="false" type="String">
+		<Option label="%bindAuthMechanism.none" value="none" />
+		<Option label="%bindAuthMechanism.simple" value="simple" />
+		<Option label="%bindAuthMechanism.GSSAPI" value="GSSAPI" />
+	</AD>
+
+	<AD id="krb5Principal" name="internal" description="internal use only" type="String" required="false" />
+	<AD id="krb5TicketCache" name="internal" description="internal use only" type="String" required="false" />
```

Comment:
We should use beta instead of internal for all the LDAP support Kerberos new metatype

Reply:
Will set to beta when we deliver the runtime code.